apps/mobile/src/queries/balance/btc-balance.query.ts

```diff
@@ -5,11 +5,18 @@ import { QueryFunctionContext, useQuery } from '@tanstack/react-query';
 
 import { AccountRequest, getBtcBalancesService } from '@leather.io/services';
 
+/**
+ * @deprecated useBtcTotalBalance is not used now we have moved to single account view
```

Comment:
On one hand these are still going to be useful at some point, on the other hand I see how this can be mistakenly used. I think it's worth rethinking the data hook design at some point in a way that solves both.